src/main/java/mat/server/util/MeasureUtility.java

```diff
@@ -69,6 +69,15 @@ public static String formatVersionText(String revisionNumber, String version) {
         return formatVersionText(version) + "." + revisionFormat.format(Integer.parseInt(revisionNumber));
     }
 
+    public static String formatVersionTextForIncludeLibraries(String version) {
+        StringUtility su = new StringUtility();
+        String[] versionArr = version.split("\\.");

```

Comment:
should we handle safely cases when a version value has less than 3 or even 0 elements?

Reply:
Yes, working on it.